src/linagora.esn.calendar/app/components/calendars-list/user/user-calendars-list.pug

```diff
@@ -2,16 +2,16 @@
   .lv-item.calendar-item.calendar-header-title.hidden-md
     .media
       .pull-left
-        i.mdi.mdi-calendar
+        i.mdi.mdi-checkbox-blank-circle.all-calendars-selected
```

Comment:
please use `checkbox-multiple-blank-circle` icon like it's describe in the issue https://github.com/linagora/calendar-contacts-private/issues/31